Rework Installation into Distribution Channels (#131)
DevExpress/devextreme-documentation: concepts/Common/Distribution Channels/15 ZIP Archive.md

> The ZIP archive is a suitable choice if you [use jQuery](/concepts/58%20jQuery%20Components/05%20Add%20DevExtreme%20to%20a%20jQuery%20Application/10%20Local%20Files.md '/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/#Local_Files'), develop applications with DevExtreme on Mac or Linux, or simply want to try DevExtreme. The archive contains DevExtreme scripts, stylesheets, and demos.

Reviewer: ```suggestion
The ZIP archive can be used if you [use jQuery](/concepts/58%20jQuery%20Components/05%20Add%20DevExtreme%20to%20a%20jQuery%20Application/10%20Local%20Files.md '/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/#Local_Files'), develop applications with DevExtreme on Mac or Linux, or simply want to try DevExtreme. The archive contains DevExtreme scripts, stylesheets, and demos.
```
